packages/shorebird_cli/lib/src/commands/flutter/flutter_command.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:shorebird_cli/src/command.dart';
+
+/// {@template flutter_command}
+/// `shorebird flutter`
+/// Manage your Shorebird Flutter installation.
+/// {@endtemplate}
+class FlutterCommand extends ShorebirdCommand {
+  /// {@macro flutter_command}
+  FlutterCommand();
+
+  @override
+  String get description => 'Manage your Shorebird Flutter installation.';
+
+  @override
+  String get name => 'flutter';
```

Comment:
Maybe we hide this until it does something?

Reply:
It should do something in the next few commits. I'll hide it if we end up releasing before we add useful functionality.